File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGFlexDirection.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGFlexDirection {
    YGFlexDirectionColumn,
    YGFlexDirectionColumnReverse,
    YGFlexDirectionRow,
    YGFlexDirectionRowReverse;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGFlexDirection forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGFlexDirection.kt
package app.cash.redwood.yoga.internal.enums

enum class YGFlexDirection {
    YGFlexDirectionColumn, YGFlexDirectionColumnReverse, YGFlexDirectionRow, YGFlexDirectionRowReverse;

    fun getValue(): Int {
        return ordinal
    }

    companion object {

        fun forValue(value: Int): YGFlexDirection {
            return values()[value]
        }
    }
}
